stock_picking_quick/views/picking_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <field name="move_ids_without_package" position="before">
+                <button
+                    name="add_product"
+                    string="Add"
+                    type="object"
+                    attrs="{'invisible': [('state', 'not in', ('draft', 'waiting'))]}"
```

Comment:
assigned state should support visible button

Reply:
Should confirmed state also display the button ?

Reply:
`ADD` button should be visible in the same conditions than `Add a line` I suppose, then also confirmed.